### ISSUE TITLE
[OPFOR] Kill command does not work

### ISSUE THREAD

**Issue by Nevermore2790** [ValveSoftware/halflife#1587]
Typing kill in the console supposed to kill you instantly. However, in Opposing Force, typing kill in the console results in nothing.

**Comment by treetoon** [comment]
Can confirm.

**Comment by Nevermore2790** [comment]
What do you mean?? I've tried this since before it was SteamPiped up to now, and it results in nothing unlike in Half-Life 1 and Blue Shift where typing this kills you instantly.  Are you aware of this? Thanks.

**Comment by treetoon** [comment]
Yes, which is why I could confirm this as I tried it myself and it didn't work.

**Comment by tschumann** [comment]
Which platform(s)?

**Comment by SamVanheer** [comment]
This happens because for some reason the kill command checks if you're playing a CTF game (it checks if you have a non-zero team number).
